.github/workflows/main.yml

```diff
@@ -110,7 +110,7 @@ jobs:
       run: |
         cd build
         mv regression surelog-linux-${{matrix.compiler}}-regression
-        find surelog-linux-${{matrix.compiler}}-regression -name "*.tgz" | tar czfp surelog-linux-${{matrix.compiler}}-regression.tgz -T -
+        find surelog-linux-${{matrix.compiler}}-regression -name "*.tar.gz" | tar czfp surelog-linux-${{matrix.compiler}}-regression.tgz -T -
```

Comment:
the file that is packed up then is named `*.tgz` again, wouldn't we want it name `*.tar.gz` as well ?

Reply:
Those are not archived using python. They are archived using GitHub action.

Reply:
Yes, but *.tar.gz is the correct naming anyway. *.tgz is a very uncommon way to name such archives.
Also, uniformly names are also beneficial as it leaves people less scratching their heads.

Reply:
Updated all references of .tgz to .tar.gz.